### DISCUSSION TITLE
Unity ECS实践 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#24] [Announcements]
# Unity ECS实践 - 没有秃头基因的燊的博客

稍微聊聊ECS架构并简单罗一下Unity中的ECS的几个步骤 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/47772/

**Comment by yzs020220** [comment]
Windows文件不区分大小写，上传上来网站又区分了，图片路径看了半天 :poop:
